### PR TITLE
app notification docs

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: docs/guides/cloud/recorded-runs.mdx [hunk 1]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
 
 The [Debug](/guides/core-concepts/cypress-app#Debug) page allows you to easily
 debug failed CI test runs from within the Cypress app. From this page, you can
-review, rerun, and debug failed CI test runs that are recorded to Cypress
+review, rerun, and debug faƒiled CI test runs that are recorded to Cypress
```
Comment (MikeMcC399) [inline docs/guides/cloud/recorded-runs.mdx:373]:
```suggestion
review, rerun, and debug failed CI test runs that are recorded to Cypress
```

File: docs/guides/cloud/recorded-runs.mdx [hunk 2]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
...
 />
+
+### Notifications
+
+Cypress Cloud can notify you about recorded test runs on your branch directly in
+the app through system notifications.
+
+<DocsImage
+  src="/img/guides/cloud/runs/app-notification.jpg"
+  alt="App notification on MacOS"
```
Comment (MikeMcC399) [inline docs/guides/cloud/recorded-runs.mdx:388]:
```suggestion
  alt="App notification on macOS"
```

File: docs/guides/cloud/recorded-runs.mdx [hunk 3]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
...
+/>
+
+Clicking the notification will take you to the [debug](#Debug) page in the
+Cypress app.
+
+:::info
+
+In-app notifications are available in Cypress v12.15.0 and above and requires
+MacOS or Linux. Windows support is coming soon.
```
Comment (MikeMcC399) [inline docs/guides/cloud/recorded-runs.mdx:397]:
```suggestion
macOS or Linux. Windows support is coming soon.
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 4]
```diff
@@ -204,6 +204,17 @@ environment setups (e.g. `docker`/`docker-compose`), you may have to pass git
 information using
 [custom environment variables](/guides/continuous-integration/introduction#Git-information).
 
+### Cloud Run Notifications
+
+<DocsImage
+  src="/img/guides/cloud/runs/app-notification.jpg"
+  alt="App notification on MacOS"
```
Comment (MikeMcC399) [inline docs/guides/core-concepts/cypress-app.mdx:211]:
```suggestion
  alt="App notification on macOS"
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Mostly "MacOS" => "macOS"

see https://support.apple.com/macos

File: docs/guides/cloud/recorded-runs.mdx [hunk 5]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
...
+<DocsImage
+  src="/img/guides/cloud/runs/enable-notifications.jpg"
+  alt="Enable notifications"
+/>
+
+You can receive notifications for the following events:
+
+- When a recorded run starts on your current commit in Cypress Cloud
+- When a recorded run starts to fail on your current commit in Cypress Cloud
+- And when a recorded run completes on your current commit in Cypress Cloud,
```
Comment (jaffrepaul) [inline docs/guides/cloud/recorded-runs.mdx:416]: Is this the only notification type where the action can be customized?

Comment (jaffrepaul) [reply docs/guides/cloud/recorded-runs.mdx:416]: nm, I see it in the screenshot 😅

File: docs/guides/cloud/recorded-runs.mdx [hunk 6]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
...
+:::
+
+Click the **Send test notification** button to test that notifications are
+working properly on your system.
+
+#### Troubleshooting Notifications
+
+If you are not receiving notifications, please check the following:
+
+- That the recorded run is being recorded to Cypress Cloud (This typically only
```
Comment (jaffrepaul) [inline docs/guides/cloud/recorded-runs.mdx:434]: Just to clarify, is this saying that you may not get notifications if the run isn't being recorded in CI. So you would normally get them locally?

**Review by jaffrepaul — CHANGES_REQUESTED** [review]
Added a couple of comments/clarifying questions.

File: docs/guides/cloud/recorded-runs.mdx [hunk 7]
```diff
@@ -370,10 +370,70 @@ information about specific runs can be found on the
...
+  happens in CI and not locally).
```
Comment (elylucas) [inline docs/guides/cloud/recorded-runs.mdx:435]:
```suggestion
- That the recorded run is being recorded to Cypress Cloud.
```

**Comment by elylucas** [comment]
> Mostly "MacOS" => "macOS"
> 
> see https://support.apple.com/macos

Thanks much!

**Review by MikeMcC399 — COMMENTED** [review]
My previous comments  have all been addressed.
Many thanks 👍🏻

**Review by MikeMcC399 — APPROVED** [review]

**Review by jaffrepaul — APPROVED** [review]

**Comment by lmiller1990** [comment]
I also wrote this same content in: https://github.com/cypress-io/cypress-documentation/pull/5327

Shall we merge these together? I think both has useful content.

**Pull request merged**